src/test/java/ua/lviv/iot/algo/part1/lab3/HelicopterTest.java

```diff
@@ -0,0 +1,65 @@
+package ua.lviv.iot.algo.part1.lab3;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+class HelicopterTest {
+    private final Helicopter helicopter = new Helicopter(7000, 13000,
+            "Mi", 70,
+            101, "Hip",
+            2000, 3800,
+            2000, 200,
+            800);
+    private final Helicopter helicopter_null = new Helicopter();
+
+    @Test
+    public void testTakeOff() {
+        int newCurrentAltidute = helicopter.getCurrentAltitude() + 100;
+        helicopter.takeOff();
+        assertEquals(newCurrentAltidute, helicopter.getCurrentAltitude());
+    }
+
+    @Test
+    public void testAscend() {
+        int newCurrentAltidute = helicopter.getCurrentAltitude() + 200;
+        helicopter.ascend(200);
+        assertEquals(newCurrentAltidute, helicopter.getCurrentAltitude());
+        // ascend further than maxAltidute
```

Comment:
The more isolated test is - the better. This would better be divided into 2 tests